src/main.ts

```diff
@@ -54,6 +64,10 @@ async function run(): Promise<void> {
       });
 
       setOutput(ref.output, value);
+
+      if (setEnv) {
+        exportVariable(ref.output, value);
+      }
```

Comment:
Since there's some branching logic here, we may want to add `logDebug` messages when the output is being set and when the variable is being exported.

Reply:
It is unclear what you are asking for here. I searched for `logDebug` in the project, but got no results. It doesn't seem essential to me, but if you think it is necessary, could you please clarify, or perhaps add it yourself?